### ISSUE TITLE
Lab 03 - Part 2 - Measuring Gentrification

### ISSUE THREAD

**Issue by Niagara1000** [DS4PS/cpp-528-spr-2021#17]
Hi @cenuno ,

I have attached a picture of the part of the Lab 03 Tutorial that is very confusing to me (it's the red box in the picture). 

In the tutorial, inflation in 2000 is taken into account when we are studying 2000 and 2010 data. 

However, for Lab 03, the instructions say that we will work on the years 1990 and 2000. 

So, do we need to rewrite the code as follows? : 

```
mhv.90 <- d$mhmval90
mhv.00 <- d$mhmval00
```

I'm also not sure if I should multiply `mhv.00` in the code above by `1.28855`..

<br><br>

<img width="984" alt="Screen Shot 2021-03-25 at 11 09 54 AM" src="https://user-images.githubusercontent.com/46457863/112523387-f365af80-8d5b-11eb-91cd-9d3e5a74ae61.png">

<br><br>

Thank you!

-Archana

**Comment by cenuno** [comment]
You are correct! Good catch seeing the need to update which variables are used.

You’re also correct about the inflation rate. You want the two values to be the same so you need to take into account inflation. you’d want to “inflate” the 90s $$ values to their 00s rate.

—
Cristian E. Nuno
________________________________
From: Niagara1000 ***@***.***>
Sent: Thursday, March 25, 2021 11:23:05 AM
To: DS4PS/cpp-528-spr-2021 ***@***.***>
Cc: Cristian Ernesto Nuno ***@***.***>; Mention ***@***.***>
Subject: [DS4PS/cpp-528-spr-2021] Lab 03 - Part 2 - Measuring Gentrification (#17)


Hi @cenuno<https://github.com/cenuno> ,

I have attached a picture of the part of the Lab 03 Tutorial that is very confusing to me (it's the red box in the picture).

In the tutorial, inflation in 2000 is taken into account when we are studying 2000 and 2010 data.

However, for Lab 03, the instructions say that we will work on the years 1990 and 2000.

So, do we need to rewrite the code as follows? :

mhv.90 <- d$mhmval90
mhv.00 <- d$mhmval00


I'm also not sure if I should multiply mhv.00 in the code above by 1.28855..



[Screen Shot 2021-03-25 at 11 09 54 AM]<https://user-images.githubusercontent.com/46457863/112523387-f365af80-8d5b-11eb-91cd-9d3e5a74ae61.png>



Thank you!

-Archana

**Comment by lecy** [comment]
If you search for inflation calculators you can find the proper scaling factor for those periods: 

https://www.usinflationcalculator.com/

![image](https://user-images.githubusercontent.com/1209099/112526957-c1eee300-8d5f-11eb-8208-bb8096df47fe.png)

**Comment by lecy** [comment]
![image](https://user-images.githubusercontent.com/1209099/112527146-f498db80-8d5f-11eb-9a8b-b3198667f7cb.png)

**Comment by Niagara1000** [comment]
> You are correct! Good catch seeing the need to update which variables are used. You’re also correct about the inflation rate. You want the two values to be the same so you need to take into account inflation. you’d want to “inflate” the 90s $$ values to their 00s rate. 

Hi @cenuno ,  thank you for your response!

so the code should be the following, correct?
```
mhv.90 <- d$mhmval90 *  1.28855
mhv.00 <- d$mhmval00
```

**Comment by cenuno** [comment]
Your inflation rate is not correct; it’s copy and pasted from above. Follow the link Dr. Lecy provided you above for a resource to obtain a rate that is accurate for 1990 to 2000.

—
Cristian E. Nuno
________________________________

**Comment by Niagara1000** [comment]
Hi Dr. @lecy ,

Thank you for your response

I am not sure about what the following bolded text means from the screenshot below

"
 Otherwise, you can always estimate it if you know the average long-term inflation rate. 
**Since 2000, the rate has averaged about 2.5% in the US**, so you can use: 

1. so does this sentence refer to the time period of **2000 - 2021**?
2. Also, how did we get the number 1.025 from 2.5%? It looks like we need to divide by 2, but not sure why

```
# 10-year inflation factor
(1.025)^10

> [1] 1.280085
```
"

I am unsure of how to use this logic for the time period from 1990 to 2000


<br><br>

![](https://user-images.githubusercontent.com/1209099/112527146-f498db80-8d5f-11eb-9a8b-b3198667f7cb.png)

**Comment by cenuno** [comment]
Hi Archana,

We can talk more about this in office hours tonight. Whenever you take a raise a number between zero and one to a positive non-one power, you always get back something smaller.

If you were to write 0.025^10, you’d get something even closer to zero than 2.5%. To account for this, I think Dr. Lecy added 1 to get 1.025.

When you add one, you now escape this issue of having a number between zero and one being raised by a positive non one power returning back a number very close to zero.

This data transformation is a small and subtle point but actually very meaningful. With it, we can now ask “what is the average inflation rate over the course of a decade?” by adding one to the average inflation rate 2.5% and raising to the power of ten (one year in a decade).


—
Cristian E. Nuno
________________________________

**Comment by malmufre** [comment]
Hello,

I actually have the same question as Archana. I used the https://westegg.com/inflation/ calculator and I got :
What cost $1 in 1900 would cost $20.63 in 2000.
I have used the same logic that was used to understand the inflation rate from 2000 to 2010 which by the following:
```
calc2010<-(1.25-1)/1*100

calc2010<-calc2010/10

calcl2010<-1+0.025

1.025^10

```
when I used the same logic for the change from 1990 to 2000 I got a very big number so I was wondering if it's correct:
```
calc2000<-(20.63-1)/1*100

calc2000<-calc2000/10   #196.3% per year

calc2000<-1+1.963

2.963^10
```

This  will return 52,157.47 , does this mean that I have to multiply 
`mhv.90 <- d$mhmval90 * 52,157.47`
Because it seems to me that it is an exaggerated number.

**Comment by cenuno** [comment]
Hi Marah,

You might have mistakenly used "2020" as the end year rather than "2000".

When I set the initial year as 1990 and end year as 2000 within the westegg website here is what I get back:

```quote
The Inflation Calculator
What cost $1 in 1990 would cost $1.31 in 2000.
Also, if you were to buy exactly the same products in 2000 and 1990,
they would cost you $1 and $0.75 respectively.

Do you want to do another calculation?
```

Your intuition is correct: the inflation rate you are calculating is way too large. Check out the historical rates that confirm your intuition: https://fred.stlouisfed.org/series/FPCPITOTLZGUSA.

**Comment by malmufre** [comment]
Thank you for the correction.
I rechecked on westegg website and I will attach a screenshot of what I got.
![image](https://user-images.githubusercontent.com/59895354/112727764-18614a80-8f35-11eb-8998-10365d29fac8.png)
I will use the number you have provided because for some reason I am not getting the same values as you even when setting the initial year as 1990 and end year as 2000.

**Comment by cenuno** [comment]
~That is so interesting! Westegg must use some sort of geolocator that checks where the request is coming from and uses that location's currency to represent this number.~

~I have confidence that my results are in USD but I am not sure about your results. That is first time this has happened in this course.~

Nope I was wrong haha. I totally missed the 1900 part! Thank you Marah for sharing, I assure you others will do the same. Thank you!

**Comment by malmufre** [comment]
Hi Prof 

You are absolutely right as I was setting the initial year as 1900 by mistake. 
Thank you for the correction.
